spec/Section 3 -- Type System.md

```diff
@@ -2164,3 +2164,128 @@ to the relevant IETF specification.
 ```graphql example
 scalar UUID @specifiedBy(url: "https://tools.ietf.org/html/rfc4122")
 ```
+
+## Schema Coordinates
+
+SchemaCoordinate :
+
+- Name
+- Name . Name
+- Name . Name ( Name : )
+- @ Name
+- @ Name ( Name : )
+
+:: A _schema coordinate_ is a human readable string that uniquely identifies a
+_schema element_ within a GraphQL Schema.
+
+:: A _schema element_ is a specific instance of a named type, field, input
+field, enum value, field argument, directive, or directive argument.
+
+A _schema coordinate_ is always unique. Each _schema element_ may be referenced
+by exactly one possible schema coordinate.
+
+A _schema coordinate_ may refer to either a defined or built-in _schema
+element_. For example, `String` and `@deprecated(reason:)` are both valid schema
+coordinates which refer to built-in schema elements. However it must not refer
+to a meta-field. For example, `Business.__typename` is _not_ a valid schema
+coordinate.
```

Comment:
Is there a specific reason to forbid meta fields? I can definitely see myself using it. For an example to log introspection usages of my server.

Reply:
Good question! I don't actually remember if there was a technical reason, or if was around limiting the spec to the intended application use cases 🤔 

This [blames](https://github.com/graphql/graphql-spec/commit/b65eb313af7a844d243473743881412cad049014) to @leebyron, so ping Lee to see if he remembers.

Reply:
@leebyron any thoughts?